app/views/extended_metadata_types/_extended_metadata_type_table_row.html.erb

```diff
@@ -0,0 +1,19 @@
+<tr class="<%= 'warning' if extended_metadata_type.disabled_but_used? %>">
+  <td><%= extended_metadata_type.id %></td>
+  <td><%= extended_metadata_type.title %></td>
+  <td><%= extended_metadata_type.supported_type %></td>
+  <td><%= extended_metadata_type.extended_metadatas.count %></td>
+  <td><%= extended_metadata_type.enabled? ? 'Enabled' : 'Disabled' %></td>
+  <td>
+    <% unless extended_metadata_type.extended_type? %>
+      <%= form_for(extended_metadata_type, url: administer_update_extended_metadata_type_path(extended_metadata_type), method: :put) do |f| %>
+        <%= f.hidden_field :enabled, value: !extended_metadata_type.enabled? %>
+        <% if extended_metadata_type.enabled? %>
+          <%= f.submit 'Disable', class:'btn' %>
```

Comment:
maybe could add some color for the buttons

```
     <%= f.submit 'Disable', class:'btn btn-danger' %>
        <% else %>
      <%= f.submit 'Enable', class:'btn btn-success' %>
```

Reply:
I did wonder about that, but maybe btn-warning for Disable, danger seems a bit scarey

Reply:
It would be good just to distinguish two statuses in any way.